libraries/AP_DDS/README.md

```diff
@@ -377,3 +377,40 @@ This will run the tools automatically when you commit. If there are changes, jus
   pre-commit install
   git commit
   ```
+
+## Testing DDS on Hardware
+
+### With Serial
+
+The easiest way to test DDS is to make use of some boards providing two serial interfaces over USB such as the Pixhawk 6X.
+The [Pixhawk6X/hwdef.dat](../AP_HAL_ChibiOS/hwdef/Pixhawk6X/hwdef.dat) file has this info.
+```
+SERIAL_ORDER OTG1 UART7 UART5 USART1 UART8 USART2 UART4 USART3 OTG2
+```
+
+For example, build, flash, and set up OTG2 for DDS
+```bash
+./waf configure --board Pixhawk6X --enable-dds
+./waf plane --upload
+mavproxy.py --console
```

Comment:
Do you need to specify the device, or is this automatic? On macOS I would have to add `--master /dev/cu.usbserial-0001` or similar.

Reply:
On Linux, it's automatic right now with USB.